.changeset/eleven-cougars-film.md

```diff
@@ -0,0 +1,15 @@
+---
+"@clerk/react-router": patch
+"@clerk/shared": minor
```

Comment:
With the introduction of `es2020` target, would it make sense to publish a minor version?

Reply:
Seems fine 👍